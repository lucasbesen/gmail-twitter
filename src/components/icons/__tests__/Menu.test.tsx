import React from 'react';
import renderer from 'react-test-renderer';

import { Menu } from '../';

it('should render Menu icon without error', () => {
  const component = renderer.create(<Menu />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
