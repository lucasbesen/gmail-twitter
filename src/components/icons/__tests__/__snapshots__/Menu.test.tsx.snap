// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Menu icon without error 1`] = `
<svg
  aria-hidden="true"
  className="sc-bxivhb gpbNsD StyledIconBase-sc-bdy9j4 jbPeIR"
  fill="currentColor"
  focusable="false"
  viewBox="0 0 24 24"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
  />
</svg>
`;
